src/magazine.py

```diff
@@ -26,9 +27,15 @@ def __init__(self, sheet_row, publication):
         self.publication = publication
 
     def download_magazine(self, id):
+        """
+        pdf response and first image response from appdev upload service
+        
+        returns response tuple (<pdf_response>, <first_page_response>):
+        ({"success": <bool>, "data": "<pdf url?>"}, {"success": <bool>, "data": "<first page png url?>"})
+        """
         creds = service_account.Credentials.from_service_account_file(
-            os.getenv("GOOGLE_SERVICE_ACCOUNT_PATH")
-        )
+            os.getenv("GOOGLE_SERVICE_ACCOUNT_PATH"))
```

Comment:
i think the parenthesis belong in the same indentation as the `creds = ...` line so this line's change should be reverted